packages/date-time-utilities/package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "@fluentui/date-time-utilities",
+  "version": "7.0.0",
+  "description": "Date-time utilities",
+  "main": "lib-commonjs/index.js",
+  "module": "lib/index.js",
+  "typings": "lib/index.d.ts",
+  "sideEffects": [
+    "lib/version.js"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/microsoft/fluentui"
+  },
+  "license": "MIT",
+  "scripts": {
+    "build": "just-scripts build",
+    "bundle": "just-scripts bundle",
```

Comment:
Probably remove this since the package doesn't have a webpack config (and I don't think it really needs one)
```suggestion
```